com.github.UmairKhurshid.CaesarCipher/com.github.UmairKhurshid.CaesarCipher.yaml

```diff
@@ -0,0 +1,27 @@
+id: com.github.UmairKhurshid.CaesarCipher
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: caesar-cipher
+tags:
+  - CLI
+  - Utilities
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --no-display
```

Comment:
what does that `no-display` mean and which bit of the documentation told you this (so we can fix it)?